docs/backuprestore-with-oadp.md

```diff
@@ -256,6 +258,10 @@ status:
     type: Reconciled
 ```
 
+> [!NOTE]
+> When LVMS is used to create PVs dynamically in the cluster, please ensure that the CSI Integration (i.e., 
```

Comment:
We can probably do that by default, would not be needed by lso either

Reply:
gotcha, maybe @jakobmoellerdev can help us confirm this

(keeping the note for now, if / when confirmed I may drop the note - if nothing against it)

Reply:
I think restic is only necessary in case we want to copy over the data of the PV into the backup itself. For IBU this should never be necessary since the data is kept on disk and reused after pivot.